tasks/cron.yml

```diff
@@ -0,0 +1,8 @@
+- name: Update topologies cron
```

Comment:
add:

```yaml
when: openwisp2_network_topology
```

otherwise the cron will be installed also when the module is not active.